Okta.AspNet.Abstractions/OktaWebOptions.cs

```diff
@@ -36,5 +36,10 @@ public class OktaWebOptions
         /// The clock skew.
         /// </value>
         public TimeSpan ClockSkew { get; set; } = DefaultClockSkew;
+
+        /// <summary>
+        /// Gets or sets the uri of your organization's proxy server.  The default is null.
+        /// </summary>
+        public ProxyConfiguration Proxy { get; set; }
```

Comment:
Have you double-checked if this should be excluded for Web API?

Reply:
I'll review that point further today.

Reply:
Reviewed and tested this more closely this setting should NOT be excluded from Web API; if the consumer has a proxy server and requires proxy settings, that will affect Mvc and WebApi.